scripts/install_librocksdb.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+ROCKSDB_VERSION="8.5.3"
+
+# Check if RocksDB is already installed
+if [[ $(find /usr/lib -name "librocksdb.so.${ROCKSDB_VERSION}" -print -quit) ]]; then
+    read -r -p "RocksDB version ${ROCKSDB_VERSION} is already installed. Do you want to reinstall it? (yes/no): " choice
+    case "$choice" in
+    y | yes | Yes | YES)
+        echo "Reinstalling RocksDB..."
+        rm -rf /usr/lib/librocksdb*
+        ;;
+    n | no | No | NO)
+        echo "Skipping RocksDB installation."
+        exit 0
+        ;;
+    *)
+        echo "Invalid choice. Please enter 'yes' or 'no'."
+        exit 1
+        ;;
+    esac
+else
+    # RocksDB is not installed, proceed with installation
+    echo "RocksDB is not installed. Proceeding with installation..."
+fi
+
+# Check the OS type and perform different actions
+if [[ $(uname) == "Linux" ]]; then
+    # Check Linux distribution
+    if [[ -f /etc/os-release ]]; then
+        source /etc/os-release
+
+        if [[ "$ID" == "ubuntu" ]]; then
+            # Ubuntu specific dep installation
+            echo "Installing RocksDB dependencies..."
+            apt-get install libgflags-dev libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev build-essential clang
+
+        elif [[ "$ID" == "alpine" ]]; then
+            # Alpine specific dep installation
+            echo "Installing RocksDB dependencies..."
+            # 1. Install dependencies
+            echo "@testing http://nl.alpinelinux.org/alpine/edge/testing" >>/etc/apk/repositories
+            apk add --update --no-cache cmake bash perl g++
+            apk add --update --no-cache zlib zlib-dev bzip2 bzip2-dev snappy snappy-dev lz4 lz4-dev zstd@testing zstd-dev@testing libtbb-dev@testing libtbb@testing
+            # 2. Install latest gflags
+            cd /tmp &&
+                git clone https://github.com/gflags/gflags.git &&
+                cd gflags &&
+                mkdir build &&
+                cd build &&
+                cmake -DBUILD_SHARED_LIBS=1 -DGFLAGS_INSTALL_SHARED_LIBS=1 .. &&
+                make install &&
+                rm -rf /tmp/gflags
+        else
+            echo "Linux distribution not supported"
+            exit 1
+        fi
+
+        # 3. Install Rocksdb (same for any linux distribution)
```

Comment:
Can't we have some Prebuilt RocksDB binaries instead of building from source during the Docker build ? I think this can significantly reduce the build time.

Reply:
The asset looks available in https://github.com/facebook/rocksdb/releases/tag/v8.5.3

Reply:
Those assets are just the source code, don't think will improve the build time